PLog.cs

```diff
@@ -1,13 +1,61 @@
-﻿using System;
-
-namespace p2pcopy
-{
-	public class PLog
-	{
-		public static void DEBUG(string fmt, params object[] args)
-		{
-			//Console.WriteLine(string.Format(fmt, args));
-		}
-	}
-}
-	
\ No newline at end of file
+﻿using System;
+
+namespace p2pcopy
+{
+    public class PLog
+    {
+        private static bool debug = false;
+        private static bool verbose = false;
+
+        public static bool Debug
+        {
+            get
+            {
+                return debug;
+            }
+
+            internal set
+            {
+                debug = value;
+            }
+        }
+            
+        public static bool Verbose
+        {
+            get
+            {
+                return verbose;
+            }
+
+            internal set
+            {
+                verbose = value;
+            }
+        }

```

Comment:
nitpick: have you heard about C# auto-properties? I guess you can use them here so that you don't need the backing fields

Reply:
Sure will do that :+1: